.eleventy.js

```diff
@@ -78,6 +79,15 @@ module.exports = function (eleventyConfig) {
     });
   });
 
+  eleventyConfig.addShortcode("lastUpdated", function (component) {
+    const dirPath = path.join(__dirname, "src/moj/components", component);
+    const lastUpdated = execSync(
+      `LANG=en_GB git log -n1 --pretty=format:%ad --date=format:'%e %B %Y' ${dirPath}`
```

Comment:
Fun fact: I had to specify the language because otherwise it does it in your system's language 🇪🇸